ports/atmel-samd/common-hal/busio/SPI.c

```diff
@@ -25,25 +25,35 @@
  */
 
 #include "shared-bindings/busio/SPI.h"
-#include "py/nlr.h"
+#include "py/mperrno.h"
 #include "py/runtime.h"
-#include "rgb_led_status.h"
-#include "samd21_pins.h"
-#include "shared_dma.h"
 
-// We use ENABLE registers below we don't want to treat as a macro.
-#undef ENABLE
+#include "hpl_sercom_config.h"
+#include "peripheral_clk_config.h"
+
+#include "hal/include/hal_gpio.h"
+#include "hal/include/hal_spi_m_sync.h"
+#include "hal/include/hpl_spi_m_sync.h"
+
+#include "peripherals.h"
```

Comment:
This change has two _peripherals.h files. Did you mean to merge them?

Reply:
Hi - `peripherals.h` includes either `samd21_peripherals.h` or `samd51_peripherals.h`. They are slightly different. Is that what you mean?

Reply:
I don't see `peripherals.h` in the diff.

Reply:
I added it in the previous PR when I did I2C. It hasn't changed: it's just conditional `#include`s.